<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="builders.io.bank.shared.infrastructure.bus.event.virtualmachine.VirtualMachineConsumerTest" tests="2" skipped="0" failures="0" errors="0" timestamp="2023-02-24T19:28:57" hostname="MacBook-Pro-de-autentia-2.local" time="0.027">
  <properties/>
  <testcase name="accept()" classname="builders.io.bank.shared.infrastructure.bus.event.virtualmachine.VirtualMachineConsumerTest" time="0.025"/>
  <testcase name="acceptError_shouldReturnInterruptedException()" classname="builders.io.bank.shared.infrastructure.bus.event.virtualmachine.VirtualMachineConsumerTest" time="0.002"/>
  <system-out><![CDATA[2023-02-24T20:28:57.682+01:00  WARN 19080 --- [    Test worker] b.i.b.s.i.b.e.v.VirtualMachineConsumer   : executeTransactionOnTransactionSent interrupted!: null
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
